studio/src/app/types/core/settings.ts

```diff
@@ -18,4 +20,5 @@ export interface SettingsPanels {
 export interface Settings {
   panels: SettingsPanels;
   editMode: EditMode;
+  contrastWarning: ContrastWarning;
```

Comment:
@christiankozalla for the `EditMode` I use a type because I thought that in the future their might be other mode but, to display or not the contrast warning, it might remain `true` or `false`?

Therefore maybe we can use a `boolean` here, what do you think?

Reply:
Actually, I had `boolean` first, then I saw the EditMode type and changed the type. So I already was unsure about my choice of types.

So, I'll change the type of contrastWarning to `boolean` :+1:

Reply:
Haha I was unsure too 😉 Cool let's do that, let's change it to `boolean` and then  I merge your nice PR!